src/validators/Bls12381G2Jwk.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2020 - MATTR Limited
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { assertPrivateJwk, assertPublicJwk } from "./JsonWebKey";
+import { BlsCurveName, JsonWebKey } from "../types";
+
+export const assertBls12381G2PublicJwk = (jwk: JsonWebKey): boolean => {
+  // Returns false because the BLS12-381 G1 x coordinate is not the proper length
```

Comment:
This is checking for G2 the comment needs to be updated and the check below needs to be for the right length key

Reply:
updated the comment good catch. The length as far as I can tell is correct.

log base 2 (64) = 6
768 / 6 = 128
so a G2 point (768 bits) base 64 encoded should be 128 characters from what I can tell.

Does that check out for you as well or am I making a mistake on the math?